SDK/simVis/DB/Plugin.cpp

```diff
@@ -0,0 +1,56 @@
+/* -*- mode: c++ -*- */
+/****************************************************************************
+*****                                                                  *****
+*****                   Classification: UNCLASSIFIED                   *****
+*****                    Classified By:                                *****
+*****                    Declassify On:                                *****
+*****                                                                  *****
+****************************************************************************
+*
+*
+* Developed by: Naval Research Laboratory, Tactical Electronic Warfare Div.
+*               EW Modeling & Simulation, Code 5773
+*               4555 Overlook Ave.
+*               Washington, D.C. 20375-5339
+*
+* License for source code at https://simdis.nrl.navy.mil/License.aspx
+*
+* The U.S. Government retains all rights to use, duplicate, distribute,
+* disclose, or release this software.
+*
+*/
+#include "osgEarth/TileSource"
+#include "osgDB/FileNameUtils"
+#include "osgDB/Registry"
+#include <string>
+#include "DBTileSource.h"
+
+class SimSdkOSGEarthDBDriverPlugin : public osgEarth::Contrib::TileSourceDriver
```

Comment:
This code was removed in https://github.com/USNavalResearchLaboratory/simdissdk/commit/db26d5058a6a